commit 0e3207bd59ffeb895ca11d6043dc446fda922776
Author: Srinath Mannam <srinath.mannam@broadcom.com>
Date:   Sat Jul 28 20:55:15 2018 +0530

    serial: 8250_dw: Add ACPI support for uart on Broadcom SoC
    
    commit 784c29eda5b4e28c3a56aa90b3815f9a1b0cfdc1 upstream.
    
    Add ACPI identifier HID for UART DW 8250 on Broadcom SoCs
    to match the HID passed through ACPI tables to enable
    UART controller.
    
    Signed-off-by: Srinath Mannam <srinath.mannam@broadcom.com>
    Reviewed-by: Vladimir Olovyannikov <vladimir.olovyannikov@broadcom.com>
    Tested-by: Vladimir Olovyannikov <vladimir.olovyannikov@broadcom.com>
    Reviewed-by: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
    Cc: stable <stable@vger.kernel.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/tty/serial/8250/8250_dw.c b/drivers/tty/serial/8250/8250_dw.c
index 979d1e98fcb6..2826052a7e70 100644
--- a/drivers/tty/serial/8250/8250_dw.c
+++ b/drivers/tty/serial/8250/8250_dw.c
@@ -680,6 +680,7 @@ static const struct acpi_device_id dw8250_acpi_match[] = {
 	{ "APMC0D08", 0},
 	{ "AMD0020", 0 },
 	{ "AMDI0020", 0 },
+	{ "BRCM2032", 0 },
 	{ "HISI0031", 0 },
 	{ },
 };
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1032_0e3207bd59ffeb895ca11d6043dc446fda922776.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1032_0e3207bd59ffeb895ca11d6043dc446fda922776.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
